ArcGISRuntimeSDKQt_QMLSamples/Scenes/TerrainExaggeration/TerrainExaggeration.qml

```diff
@@ -0,0 +1,137 @@
+
+import QtQuick 2.6
+import QtQuick.Controls 1.4
+import QtQuick.Controls.Styles 1.4
+import Esri.ArcGISRuntime 100.3
+import Esri.ArcGISExtras 1.1
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    property real scaleFactor: System.displayScaleFactor
+
+    SceneView {
+        id: sceneView
+        anchors.fill: parent
+
+        // add a Scene to the SceneView
+        Scene {
+            // add the BasemapTopographic basemap to the scene
+            BasemapTopographic {}
+
+            // add a surface...surface is a default property of scene
+            Surface {
+                // add an arcgis tiled elevation source...elevation source is a default property of surface
+                ArcGISTiledElevationSource {
+                    url: "http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"
+                }
+                elevationExaggeration: slider1.value
+            }
+        }
+
+        Component.onCompleted: {
+            sceneView.setViewpointCamera(camera);
+        }
+    }
+
+    Camera {
+        id: camera
+        distance: 15000
+        heading: 40.0
+        pitch: 60.0
+        roll: 0.0
+
+        Point {
+            x: -119.94891542688772
+            y: 46.75792111605992
+            z: 0
+            spatialReference: SpatialReference { wkid: 4326 }
+        }
+    }
+
+    // Slider UI presentation at bottom
+    Rectangle {
+        height: 46 * scaleFactor
+        anchors {
+            horizontalCenter: parent.horizontalCenter
+            bottom: sceneView.attributionTop
+            bottomMargin: 20 * scaleFactor
+        }
+
+        // sliderCombo: A slider and text for its value
+        Row {
+            id: sliderCombo
+            anchors {
+                centerIn: parent
+            }
+            spacing: 5
+
+            Slider {
+                id: slider1
+                anchors {
+                    verticalCenter: parent.verticalCenter
+                }
+                style: SliderStyle {
+                    groove: Rectangle {
+                        implicitWidth: 200 * scaleFactor
+                        implicitHeight: 8 * scaleFactor
+                        color: "gray"
+                        opacity: 0.7
+                        radius: 8 * scaleFactor
+                    }
+                    handle: Rectangle {
+                        anchors.centerIn: parent
+                        color: control.pressed ? "white" : "lightgray"
+                        border.color: "gray"
+                        border.width: 2 * scaleFactor
+                        implicitWidth: 34 * scaleFactor
+                        implicitHeight: 34 * scaleFactor
+                        radius: 12 * scaleFactor
+                    }
+                }
+                // Slider controls the magnitude of exaggeration
+                minimumValue: 1.0
+                maximumValue: 10.0
```

Comment:
Values can be `1` and `10` instead of `1.0` and `10.0` if we increment as integers.